clusterloader2/api/types.go

```diff
@@ -0,0 +1,83 @@
+package api
+
+import (
+	"time"
+)
+
+type Config struct {
+	// Number of namespace in the test.
```

Comment:
automanaged

Reply:
Also in general - follow the Go-style comments - they need to start with field name.

Reply:
done